apps/www/package.json

```
@@ -70,11 +93,13 @@
     "remark-code-import": "^1.1.1",
     "remark-gfm": "^3.0.1",
     "shiki": "^0.12.1",
-    "tailwindcss": "^3.1.7",
-    "typescript": "^4.5.3",
+    "storybook": "^7.0.0-beta.33",
```

```suggestion
    "storybook": "^7.0.4",
```

storybook v.7 is not out of beta. https://storybook.js.org/blog/storybook-7-0/